Frameworks/include/Starboard.h

```diff
@@ -205,4 +205,19 @@ inline CAPoint3D operator*(float f, const CAPoint3D& v) {
     return CAPoint3D(v.x * f, v.y * f, v.z * f);
 }
 
+// Wrapper for C++/WinRT types when you need a trivial default constructor
```

Comment:
[https://github.com/Microsoft/cppwinrt/blob/master/Docs/Migrating%20C%2B%2B%20CX%20source%20code%20to%20C%2B%2B%20WinRT.md](https://github.com/Microsoft/cppwinrt/blob/master/Docs/Migrating%20C%2B%2B%20CX%20source%20code%20to%20C%2B%2B%20WinRT.md)

I assume that is the topic covered in "Instantiating a C++/WinRT class".


---
In reply to: [109488370](https://github.com/Microsoft/WinObjC/pull/2382#discussion_r109488370) [](ancestors = 109488370,109485708)